macros/src/main/scala/me/rexim/morganey/meta/QuotationMacro.scala

```diff
@@ -0,0 +1,161 @@
+package me.rexim.morganey.meta
+
+import me.rexim.morganey.ast._
+
+import scala.collection.mutable.ArrayBuffer
+import scala.reflect.macros.whitebox.Context
+
+private[meta] class QuotationMacro(val c: Context) {
```

Comment:
It would be possible to test the result of a macro expansion, but I think it's impossible to run tests while expansion. I'm not quite sure, though. We'd have to try it.

If you want, you could create an issue and assign it to me, I'll add some tests for expansion-results later.